qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/JavaParameterInfo.java

```diff
@@ -21,12 +21,16 @@
  */
 public class JavaParameterInfo extends JavaElementInfo {
 
+	private static final String VARARGS = "...";

```

Comment:
Would it be better to keep this in `QuteJavaConstants.java`?

Reply:
Qute JDT has this constant but not Qute LS. So we cannot do that, because all classes from `com.redhat.qute.commons` are duplicated in Qute JDT and Qute LS. The clean solution should be have a new project `com.redhat.qute.commons` where Qute JDT and Qute LS depends on it, but it is annoying to do that with OSGi (for Qute JDT).